### DISCUSSION TITLE
how to use cp2k output

### DISCUSSION THREAD

**Discussion by deleep225** [mir-group/nequip#177] [General]
I am wondering how to use cp2k output for training.

Is xyz trajectory form the CP2K directly can be used?

In full.yaml, the xyz format is given but will it consider forces and energies while training?

I'm familiar with training vasp OUTCAR but not sure how to use cp2k trajectories with nequip.

**Comment by simonbatzner** [comment]
Hi Deleep, if ASE can read it, NequIP can read it. If ASE cannot read it, you'll have to convert it to a format ASE can read it.
